bin/copyright_header.py

```diff
@@ -0,0 +1,545 @@
+#!/usr/bin/env python3
```

Comment:
Sorry for asking you to move this file here. I think we want to keep it in the core repo for the purpose of running it as a scripted-diff.